### PR TITLE
Create a pop-up for the hamburger menu

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
## What?

I created a mobile menu pop-up for my portfolio. This pop-up contains the menu list and only appears when a user clicks the hamburger icon. It also collapses when a user clicks the close button or menu list item links.

## Why?

This is to build an organized user-friendly mobile version of my portfolio.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const iconId = document.querySelector("#hamburger-icon");
+const navId = document.querySelector('#nav-items-list-mobile');
+const listClass = document.querySelectorAll('.mobile-list-items');
+const closeButton = document.querySelector ('.close-button');
+
+
+function showToggleMenu() {
```
[the remark this answers is not in the document]
Comment (Haywayaheadshot) [reply index.js]: Thank you for the review. I have made the necessary correction and have submitted again.

File: styles.css [hunk 2]
```diff
@@ -59,6 +76,76 @@
...
+  width: 100%;
+  height: 200px;
+  height: 100%;
+  padding: 20px;
+  mix-blend-mode: multiply;
+  background-color: #6070ff;
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
+}
+
```
Comment (ichala) [inline styles.css:91]: - [ ] your website looks fantastic I would love to suggest adding a smoother animation to your website's scroll behavior, try to add `scroll-behavior: smooth;` to your HTML tag in CSS it works like magic! 😮 

``` css
html {
scroll-behavior: smooth;
}
```

Before           | After
:-------------------------:|:-------------------------:
before | mAFTER

**Review by ichala — APPROVED** [review]
Hi @Haywayaheadshot  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Good Design✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- 

- [ ] Check Comments Below


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (Haywayaheadshot) [reply styles.css:91]: Thank you @ichala I will add the suggestion before merging.

**Pull request merged**